x-pack/solutions/security/plugins/elastic_assistant/server/routes/chat/chat_complete_route.ts

```diff
@@ -114,6 +115,7 @@ export const chatCompleteRoute = (
             latestReplacements = { ...latestReplacements, ...newReplacements };
           };
 
+          const threadId = uuidv4();
```

Comment:
When this route is invoked with the same `conversationId` multiple times, is the `threadId` expected to be unique for each graph invocation?